src/components/OpenGovTreasuryProposal/index.tsx

```diff
@@ -286,27 +292,35 @@ const OpenGovTreasuryProposal = ({ className, isUsedInTreasuryTrack }: Props) =>
 		}
 	};
 
+	useImperativeHandle(ref, () => ({
+		triggerCreateProposalClick: () => {
+			handleClick();
+		}
+	}));
+
 	return (
 		<div className={className}>
-			<div
-				className={`${
-					isUsedInTreasuryTrack
-						? 'flex'
-						: 'ml-[-37px] flex min-w-[290px] cursor-pointer items-center justify-center rounded-[8px] align-middle text-[35px] text-lightBlue transition delay-150 duration-300 hover:bg-[#e5007a12] hover:text-bodyBlue dark:text-blue-dark-medium'
-				}`}
-				onClick={handleClick}
-			>
-				{isUsedInTreasuryTrack ? (
-					<CreateProposalWhiteIcon className='mr-2' />
-				) : (
-					<CreatePropoosalIcon className={`${isUsedInTreasuryTrack ? 'scale-200' : 'ml-[-31px] cursor-pointer'}`} />
-				)}
-				{isUsedInTreasuryTrack ? (
-					<p className='m-0 p-0'>Create Proposal</p>
-				) : (
-					<p className='mb-3 ml-4 mt-2.5 text-sm font-medium leading-5 tracking-[1.25%] dark:text-blue-dark-medium'>Create Treasury Proposal</p>
-				)}
-			</div>
+			{useDefaultButton && (
+				<div
+					className={`${
+						isUsedInTreasuryTrack
+							? 'flex'
+							: 'ml-[-37px] flex min-w-[290px] cursor-pointer items-center justify-center rounded-[8px] align-middle text-[35px] text-lightBlue transition delay-150 duration-300 hover:bg-[#e5007a12] hover:text-bodyBlue dark:text-blue-dark-medium'
+					}`}
+					onClick={handleClick}
+				>
+					{isUsedInTreasuryTrack ? (
+						<CreateProposalWhiteIcon className='mr-2' />
+					) : (
+						<CreatePropoosalIcon className={`${isUsedInTreasuryTrack ? 'scale-200' : 'ml-[-31px] cursor-pointer'}`} />
+					)}
+					{isUsedInTreasuryTrack ? (
+						<p className='m-0 p-0'>Create Proposal</p>
+					) : (
```

Comment:
no need to do any change in this file

Reply:
The default file has a button with it so we don't want that when using with rhs card, so I have to modify it instead of duplications